pages/app/settings.vue

```diff
@@ -51,16 +51,22 @@
             </div>
 
             <div class="col-span-4 space-y-3">
-              <div class="relative h-56 aspect-square bg-gray-800 rounded-lg">
-                <div class="absolute inset-0">
-                  <input ref="profilePhotoInput" type="file" accept="image/*" class="hidden" @change="onSelectProfilePhoto" />
-                </div>
-                <div class="flex items-center justify-center h-full">
+              <div class="relative h-56 aspect-square bg-gray-800 rounded-lg overflow-hidden">
+                <input ref="profilePhotoInput" type="file" accept="image/*" class="hidden" @change="onSelectProfilePhoto" />
+                
+                <img 
+                  v-if="imagePreview || storeUser.getProfile?.profilePicPath" 
```

Comment:
You can just use `storeUser.getProfile?.profilePicPath`. Right? I understand you want to render local user selection here. But, Let's say we only display profile pic after setting `profilePicPath` has value.